epoxy-processor/src/main/java/com/airbnb/epoxy/processor/BaseProcessorWithPackageConfigs.kt

```diff
@@ -0,0 +1,64 @@
+package com.airbnb.epoxy.processor
+
+import com.airbnb.epoxy.PackageEpoxyConfig
+import com.airbnb.epoxy.PackageModelViewConfig
+import javax.annotation.processing.RoundEnvironment
+import javax.lang.model.element.Element
+import kotlin.reflect.KClass
+
+abstract class BaseProcessorWithPackageConfigs : BaseProcessor() {
+
+    abstract val usesPackageEpoxyConfig: Boolean
+    abstract val usesModelViewConfig: Boolean
+
+    override fun supportedAnnotations(): List<KClass<*>> = mutableListOf<KClass<*>>().apply {
```

Comment:
nit: Maybe a little safer to make this final, and have an abstract `additionalSupportedAnnotations` as it seems like we can't use `@CallSuper` or something like that